docs/docs/cmd/spo/listitem/listitem-batch-add.mdx

```diff
@@ -54,6 +57,27 @@ Add a batch of items to a list retrieved by title in a specific site
 m365 spo listitem batch add --filePath "C:\Path\To\Csv\CsvFile.csv" --webUrl https://contoso.sharepoint.com/sites/project-x --listTitle "Demo List"
 ```
 
+Add a batch of items to a list retrieved by title in a specific site using csv content
+
+```sh
+$obj = $obj = @(
+  [PSCustomObject]@{
+    Title = "Item A"
+    Age = 10
+  },
+  [PSCustomObject]@{
+    Title = "Item B"
+    Age = 20
+  }
+) 
+
+$csv = $obj | ConvertTo-Csv -NoTypeInformation -Delimiter ","
+
+$csvContent = $($csv | Out-String).Replace('"','\"')
+
+m365 spo listitem batch add --csvContent $csvContent --webUrl https://contoso.sharepoint.com/sites/project-x --listTitle "Demo List"
```

Comment:
Let's simplify the part where we convert the object to CSV content. The concept stays the same but it we shorten it with a line. 

```suggestion
$obj = @(
  [PSCustomObject]@{
    Title = "Item A"
    Age = 10
  },
  [PSCustomObject]@{
    Title = "Item B"
    Age = 20
  }
) 

$csvContent = $obj | ConvertTo-Csv -NoTypeInformation -Delimiter "," | Out-String | ForEach-Object { $_.Replace('"','\"') }

m365 spo listitem batch add --csvContent $csvContent --webUrl https://contoso.sharepoint.com/sites/project-x --listTitle "Demo List"
```